qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateProcessInstanceTest.java

```diff
@@ -46,6 +47,17 @@ public void deployProcess() {
     firstProcessDefinitionKey =
         CLIENT_RULE.deployProcess(Bpmn.createExecutableProcess(processId).startEvent("v1").done());
     secondProcessDefinitionKey =
+        CLIENT_RULE.deployProcess(
+            Bpmn.createExecutableProcess(processId)
+                .eventSubProcess(
+                    "event-sub",
+                    e ->
+                        e.startEvent("msg-start-event")
+                            .message(msg -> msg.name("msg").zeebeCorrelationKey("=missing_var")))
+                .startEvent("v3")
```

Comment:
🔧 I think it's confusing to have the 2nd process definition be V3, and the 3rd process definition be V2. I'm not sure I understand the reasoning you mentioned in the commit. If it's a new deployment, how would it interfere with other tests?

Reply:
👍 I agree. I guess I just wanted to continue on what's there, but it's confusing. I'll just introduce a new process altogether.